.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - 0.10
```

Comment:
Good shout. Now testing (and passing) in 0.10, 0.12 and 4.0: https://travis-ci.org/pliersjs/pliers-svg-sprite/builds/82522140